firmware/boron-ins-fsm/test/mocks/mock_eeprom.h

```diff
@@ -1,21 +1,39 @@
+/*
+ * Mock EEPROM memory for Particle Boron
+ * Reference: https://docs.particle.io/reference/device-os/api/eeprom/eeprom/
+ * 
+ * NOTE: Due to an issue where sizeof() returns different values on different 
+ * operating systems for certain data types, this mock EEPROM implementation
+ * differs from the actual Boron EEPROM by always allocating BYTES_PER_ELEMENT
+ * bytes for each element being stored. In reality, the EEPROM only has 
+ * BORON_EEPROM_SIZE bytes of memory. When data is read from or written to the
+ * address pointed to by _address, sizeof(data) is used to read or write a
+ * number of bytes equal to the size of the data type. 
```

Comment:
I think this can be removed now with the new implementation, right?

Reply:
Thanks for reminding me, I removed it and replaced it with an updated description